CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
```

Comment:
Comment out the unreleased line for releases

Reply:
Done.